docs/reference/data-streams/tsds.asciidoc

```diff
@@ -0,0 +1,311 @@
+ifeval::["{release-state}"=="unreleased"]
+[[tsds]]
+== Time series data stream (TSDS)
+
+preview::[]
+
+A time series data stream (TSDS) models timestamped metrics data as one or
+more time series.
+
+// TODO: Replace XX% with actual percentage
+You can use a TSDS to store metrics data more efficiently. In our benchmarks,
+metrics data stored in a TSDS used XX% less disk space than a regular data
+stream.
+
+[discrete]
+[[when-to-use-tsds]]
+=== When to use a TSDS
+
+Both a <<data-streams,regular data stream>> and a TSDS can store timestamped
+metrics data. Only use a TSDS if you typically add metrics data to {es} in near
+real-time and `@timestamp` order.
+
+A TSDS is only intended for metrics data. For other timestamped data, such as
+logs or traces, use a regular data stream.
+
+[discrete]
+[[differences-from-regular-data-stream]]
+=== Differences from a regular data stream
+
+A TSDS works like a regular data stream with some key differences:
+
+* The matching index template for a TSDS requires a `data_stream` object with
+the <<time-series-mode,`index_mode: time_series`>> option. This option enables
+most TSDS-related functionality.
+
+//* In addition to a `@timestamp`, each document in a TSDS must contain one or
+//more <<time-series-dimension,dimension fields>>. The matching index template for
+//a TSDS must contain mappings for at least one `keyword` dimension.
```

Comment:
Do you mean to leave this commented out?

Reply:
That link was causing build breaks, but it's fixed now so this section is no longer commented out.